pydantic_ai_slim/pydantic_ai/_json_schema.py

```diff
@@ -26,14 +28,18 @@ def __init__(
         strict: bool | None = None,
         prefer_inlined_defs: bool = False,
         simplify_nullable_unions: bool = False,  # TODO (v2): Remove this, no longer used
+        flatten_allof: bool = False,
     ):
         self.schema = schema
 
         self.strict = strict
-        self.is_strict_compatible = True  # Can be set to False by subclasses to set `strict` on `ToolDefinition` when set not set by user explicitly
+        # Can be set to False by subclasses to set `strict` on `ToolDefinition`
+        # when not set explicitly by the user.
+        self.is_strict_compatible = True
```

Comment:
With this change, it's now ambiguous which line the comment belongs to :) Can you move it back please, even though the line is uncomfortable long? Or add a blank line between the 2 vars?